src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { BlackSwitchIcon } from "./BlackSwitchIcon";
+
+export default {
+  title: "components/BlackSwitchIcon",
```

Comment:
```suggestion
  title: "icons/BlackSwitchIcon",
```

Reply:
OK fixed ea870e5, in addition I added a `<Wrapper />` over all page's stories to make them displayed as on the application (`width: 60%` for desktop `width: 90%` for mobile)

Reply:
Besides, I created the `/icons` folder 135920b

Reply:
I created the `<StoriesContainer />` c4b0afb